draft-ppm-protocol.md

```diff
@@ -879,26 +980,23 @@ as follows:
 struct {
   TaskID task_id;
   Interval batch_interval;
-  select (protocol) { // Protocol corresponding to task_id
-    case prio: PrioCollectReq;
-    case hits: HitsCollectReq;
-  }
+  opaque output_param<0..2^16-1>;
```

Comment:
What should a collector put here in the Prio case? Empty string?

Reply:
Yes.

Reply:
Sounds fine, but I think we should have explicit text instructing implementations to do this.

Reply:
The instruction is "the leader copies the output parameter verbatim from the collect request to verify-start request". Is that not already explicitly there?

Reply:
So it is! I did notice one typo in the paragraph below. Resolving this thread.